File: JavaFundamentals/src/Arrays/Lab/EqualArrays.java
package Arrays.Lab;

import java.util.Arrays;
import java.util.Scanner;

public class EqualArrays {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] firstArr = Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int[] secondArr = Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();

        int sum = 0;
        for (int i = 0; i < firstArr.length; i++) {
            if (firstArr[i] != secondArr[i]) {
                System.out.printf("Arrays are not identical. Found difference at %d index.", i);
                return;
            }

            sum += firstArr[i];
        }

        System.out.printf("Arrays are identical. Sum: %d", sum);
    }
}

File: JS Fundamentals/Arrays - Lab/equal_arrays.js
function diff(array1, array2) {
    let diffIndex;
    for (let i = 0; i < array1.length; i++) {
        if (array1[i] !== array2[i]) {
            diffIndex = i;
            break;
        }
    }
    let result = '';
    if (diffIndex !== undefined) {
        result = `Arrays are not identical. Found difference at ${diffIndex} index`
    } else {
        let sum = eval(array1.join('+'))
        result = `Arrays are identical. Sum: ${sum}`
    }
    console.log(result);
}

diff(['10', '20', '30'], ['10', '20', '30']);
diff(['1', '2', '3', '4', '5'], ['1', '2', '4', '4', '5']);
diff(['1'], ['10']);